frontend/public/locales/en/translation.json

```diff
@@ -45,7 +45,7 @@
   "personalCriteriaScores": {
     "reasonWhyItCannotBeActivated": {
       "notLoggedIn": "You must be logged in to display personal scores.",
-      "noPersonalScore": "You have not compared this video yet."
+      "noPersonalScore": "Your ratings have not been computed yet."
     },
```

Comment:
Actually the original sentence is most often true. I propose: "You have not compared this video or your ratings have not been computer yet."